background/usercss-install-helper.js

```diff
@@ -96,28 +96,35 @@ bgReady.all.then(() => {
     const u = new URL(url);
     const m = maybeDistro[u.hostname];
     if (!m || m.rx.test(u.pathname)) {
-      openInstallerPage(tabId, url, {});
+      openInstallerPage(tabId, url, {}).catch(console.error);
       // Silently suppress navigation.
       // Don't redirect to the install URL as it'll flash the text!
-      return {redirectUrl: 'javascript:void 0'}; // eslint-disable-line no-script-url
+      return {cancel: true};
```

Comment:
Without this change, the installer is opened twice in kiwi. It seems that `redirectUrl` only works with `onBeforeRequest` or `onHeadersReceived` according to MDN:
https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/webRequest/BlockingResponse